denoiser_project/Data/transformation.py

```diff
@@ -41,4 +41,4 @@ def load_audio_files(directory, sr=16000, max_files=10, target_duration=10):
                     break
             else:
                 print(f"Skipping empty or invalid file: {file_path}")
```

Comment:
Use logging instead of print statements for better tracking or debug information.